docker/experimental/terraform/main.tf

```diff
@@ -0,0 +1,53 @@
+provider "google" {
+  project = "tpu-pytorch"
+}
+
+resource "google_artifact_registry_repository" "torch-xla-python-repo" {
+  location      = "us"
+  repository_id = "torch-xla"
+  description   = "PyTorch/XLA nightly packages"
+  format        = "PYTHON"
+}
+
+resource "google_artifact_registry_repository" "torch-xla-docker-repo" {
+  location      = "us"
+  repository_id = "torch-xla-images"
+  description   = "PyTorch/XLA nightly images"
+  format        = "DOCKER"
+}
+
+module "nightly-py37-tpuvm" {
+  source = "./modules/trigger"
+
+  release = "nightly"
+  python_version = "3.7"
+  platform = "tpuvm"
+  docker_build_args = [ "tpuvm=1" ]
```

Comment:
Does this mean that these images will build with CPP tests? Any concerns there?

Reply:
The default is still to build without the CPP tests (see the top of the Dockerfile) in this PR.